core/primitives/src/version.rs

```diff
@@ -86,6 +86,12 @@ pub const PROTOCOL_UPGRADE_SCHEDULE: Lazy<ProtocolUpgradeVotingSchedule> = Lazy:
     let protocol_version = 70;
     let datetime = ProtocolUpgradeVotingSchedule::parse_datetime("2024-08-06 10:00:00").unwrap();
     let schedule = vec![(datetime, protocol_version)];
+
+    // FixMinStakeRatio
+    let protocol_version = 71;
+    let datetime = ProtocolUpgradeVotingSchedule::parse_datetime("2024-08-18 10:00:00").unwrap();
+    let schedule = vec![(datetime, protocol_version)];
+
```

Comment:
You're overwriting the `schedule`, shouldn't the `71` upgrade be added to the list of protocol upgrades instead of overwriting it?

Reply:
Thank you! Fixed.